rotkehlchen/globaldb/updates/manager.py

```diff
@@ -676,19 +513,26 @@ def perform_update(
         with TemporaryDirectory() as tmpdirname:
             tmpdir = Path(tmpdirname)
             temp_db_name = 'temp.db'
+            # here, the global db just needs to be initialised and not
+            # configured since its already up to date with the latest data.
             temp_db_connection, _ = initialize_globaldb(
                 global_dir=tmpdir,
                 db_filename=temp_db_name,
-                sql_vm_instructions_cb=GlobalDBHandler().conn.sql_vm_instructions_cb,
-                msg_aggregator=self.msg_aggregator,
+                sql_vm_instructions_cb=self.globaldb.conn.sql_vm_instructions_cb,
             )
 
             # use a critical section to avoid modifications in the globaldb during the update
             # process since we ignore any possible change in the user globaldb once we started
             # the update.
-            with GlobalDBHandler().conn.critical_section():
+            with self.globaldb.conn.critical_section():
                 log.info('Starting assets update. Copying content from the user globaldb')
                 _replace_assets_from_db(temp_db_connection, global_db_path)
+
+                # foreign keys are turned ON to ensure that force_remote_asset() properly cascade-deletes  # noqa: E501
+                # related entries in multiasset_mappings, underlying_tokens_list, etc. when removing assets  # noqa: E501
```

Comment:
Don't use noqa in multiline comments like this. Just split it. Something like this.
```suggestion
                # foreign keys are turned ON to ensure that force_remote_asset() properly
                # cascade-deletes related entries in multiasset_mappings, underlying_tokens_list
                # , etc. when removing assets 
```